platform/viewer/cypress/integration/common/OHIFCornerstoneToolbar.spec.js

```diff
@@ -425,6 +425,56 @@ describe('OHIF Cornerstone Toolbar', () => {
     });
   });
 
+  it('checks if Eraser tool will remove the measurements added in the viewport', () => {
```

Comment:
We removed the erase tool, so you can remove this test, must have came from a dodgey merge.

Reply:
Removed, thanks.